pyi.py

```diff
@@ -20,7 +21,7 @@
     ClassScope,
     FunctionScope,
 )
-from typing import ClassVar, NamedTuple
+from typing import ClassVar, Iterator, NamedTuple
```

Comment:
Could we import this from `collections.abc`, please ;)

Reply:
No because we still support 3.7

Reply:
Yes but future annotations are enabled at the top of the file

Reply:
Right, good point